### PR TITLE
added cart page

### PR DESCRIPTION
**Pull request by KanekarSnehal**
Added cart page with following functionality
- user can add product to cart from product-listing page and also wishlist page
- once product is added to cart from product-listing page the button will change to go to cart which will redirect the user to cart.
- if the product is added to cart from wishlist once then further the quantity of that product will be increased.
- On the cart page the quantity of product can be incremented/decremented and also product can be moved to wishlist if it does not exist in wishlist else user will get the option of remove from wishlist.
- User can also directly remove the product from the cart.
- The Cart Billboard shows the Total MRP, Total discount on MRP and the Total Amount to be paid.

Files to check
- src/api-calls/addTocart
- src/api-calls/removeFromCart
- src/api-calls/incrementCartItem
- src/api-calls/decrementCartItem
- src/components/CartItems
- src/components/Cards
- src/components/Wishlistitems
- src/context/cartContext
- src/reducer/wishlistReducer
- src/reducer/wishlistActionsConstants

### CODE REVIEW

File: src/api-calls/addToCart.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { cartActionsConstants } from "../reducer/index";
+
```
Comment (Isha2806) [inline src/api-calls/addToCart.js:3]: I am sure you will remove these extra spaces where its not  necessary later while refactoring your code.

**Review by Isha2806 — APPROVED** [review]
Snehal I must say, a splendid App is in progress!  
I like how descriptive the Pr is. For UI, the color looks good to my eyes. 
you can improve more on styling later when all the functionalities are done.

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]
Everything seems to work fine, I liked how you have made small functions for each  task, such as increment  decrement items, also you have maintained constants separately

**Review by Mohitkumar33 — APPROVED** [review]
Nice and clean code with nice variable naming

**Comment by KanekarSnehal** [comment]
> Snehal I must say, a splendid App is in progress! I like how descriptive the Pr is. For UI, the color looks good to my eyes. you can improve more on styling later when all the functionalities are done.

Thank you Isha for your feedback, I will work on styling later on.

**Comment by KanekarSnehal** [comment]
> Everything seems to work fine, I liked how you have made small functions for each task, such as increment decrement items, also you have maintained constants separately

Thank you for your feedback.

**Comment by KanekarSnehal** [comment]
> Nice and clean code with nice variable naming

Thank you Mohit

**Pull request merged**